src/lib.rs

```diff
@@ -33,82 +33,51 @@
 //! ```
 //! [__Complete examples__](https://github.com/carloskiki/leptos-icons/tree/main/examples) are available on github.
 
-use leptos::*;
+use leptos::{prelude::*, svg};
 
 /// The Icon component.
 #[component]
 pub fn Icon(
     /// The icon to render.
     #[prop(into)]
-    icon: MaybeSignal<icondata_core::Icon>,
+    icon: Signal<icondata_core::Icon>,
     /// The width of the icon (horizontal side length of the square surrounding the icon). Defaults to "1em".
-    #[prop(into, optional)]
-    width: MaybeProp<TextProp>,
+    #[prop(optional, into)]
+    width: MaybeProp<String>,
```

Comment:
Why are we switching from `TextProp` to `String`?

Reply:
there was a trait impl error after the update